docs/whats-new/index.yml

```diff
@@ -27,14 +27,18 @@ landingContent:
           - text: October 2019
             url: 2019-10.md
 
-  - title: "Get involved - contribute to .NET docs"
+  - title: "Get involved - contribute"
     linkLists:
       - linkListType: overview
         links:
           - text: ".NET docs repository"
             url: https://github.com/dotnet/docs/blob/master/README.md
           - text: Project structure and labels for issues and pull requests
             url: https://github.com/dotnet/docs/blob/master/styleguide/labels-projects.md
+          - text: "Community"
```

Comment:
It feels like these two should be in a separate list. They are distinct from contributing. It's bending it a bit, but the megaphone (whatsnew) seems to fit correctly, and it's what we've used on other pages. Thoughts?